src/common/models/Projection.ts

```diff
@@ -491,6 +514,44 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityCases(
+    actualTimeseries: Array<CANActualsTimeseriesRow | null>,
+  ): Array<number | null> {
+    return actualTimeseries.map(
+      (row: CANActualsTimeseriesRow | null, i: number) => {
+        const weeklyAverage = this.getWeeklyAverageCaseForDay(i);
+        const totalPopulation = this.totalPopulation;
+        if (weeklyAverage) {
+          console.log(
+            'calc cases',
+            weeklyAverage / (totalPopulation * 0.00001),
+          );
+          return weeklyAverage / (totalPopulation * 0.00001);
```

Comment:
nit: I'd slightly prefer `(totalPopulation / 100000)` to be consistent with the PRD, etc.